.github/workflows/SPM_test_on_nidmexamples.yml

```diff
@@ -0,0 +1,46 @@
+name: SPM_on_nidm_examples              
+on: 
+  push:
+    branches: [spm_exporter]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup Python
+        uses: actions/setup-python@master
+        with:
+          python-version: 3.10.6
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest
+          pip install -r dev-requirements.txt
+          sudo apt-get install graphviz
+      - name: Test SPM parser on SPM nidm examples
+        run: |
+          mkdir -p nidmresults-examples
+          wget https://raw.githubusercontent.com/incf-nidash/nidmresults-examples/master/spm_group_ols/batch.m -O nidmresults-examples/spm_group_ols_batch.m
```

Comment:
It is not entirely clear for me which tests are computed. Here it looks like akk the spm datasets are loaded but in the `tests` folder the corresponding JSON-LD are not available. If this is not implemented yet this is fine but it would be worth clarifying in the first comment of the PR what are the tests now available ("add unitary test and CI in github actions ( compare jsonld result as graph)"--> which data are used?)

Reply:
This workflow was written to update spm parser on all all the nidmresults-examples  at each push (or pull-request if we want). The result can be directly downloaded on the artifact.  
But indeed, the name of "test" is not appropriated, because the action do not _test_ properly but simply compute spm-parser directly on all _nidmresults-examples_, in order to be checked for expert agents.

Reply:
We discussed this in our meeting on Dec 13th and agreed to add more documentation to describe what the github action is doing and to check if we can save all the examples in the repo in a future PR